### PR TITLE
Remove chart numbering being artificially inflated by 1

### PR DESCRIPTION
**Pull request by kotodharma**
I have a vague memory that we deliberately added this "1 +" in order to get numbering to show correctly when we originally created the FS stuff (I say we bec I sorta remember the 3 of us being here and working on it together when we did that :). Not sure why it seemed right then, but wrong now.

### CODE REVIEW

File: app/views/forecast_snapshots/show.html.erb [hunk 1]
```diff
@@ -229,7 +229,7 @@
           alignTicks: false
         },
         title: {
-          text: 1 + i + '. ' + '<a href="http://data.uhero.hawaii.edu/#/series?id=' + chart_data[i].id + '" target="_blank">' + chart_data[i].title + ' (' + chart_data[i].series + ')' + '</a>',
+          text: i + '. ' + '<a href="http://data.uhero.hawaii.edu/#/series?id=' + chart_data[i].id + '" target="_blank">' + chart_data[i].title + ' (' + chart_data[i].series + ')' + '</a>',
```
Comment (jonpage) [inline app/views/forecast_snapshots/show.html.erb]: Close. We need to change
`chart_data[i].id`
to
`(chart_data[i].id + 1)`

Comment (kotodharma) [inline app/views/forecast_snapshots/show.html.erb]: OK, I see where I oversimplified this and missed the fact that it was still starting from 0, but.... you didn't mean to incr the series id, did you? New version coming soon...

Comment (jonpage) [inline app/views/forecast_snapshots/show.html.erb]: You're right. But we should increment the html element id

Comment (kotodharma) [inline app/views/forecast_snapshots/show.html.erb]: OK, I think that's done by my latest changes. Pls review, and if it's still not quite right, then I'm not sure where to go next...

**Pull request merged**